src/sentry/api/paginator.py

```diff
@@ -638,11 +653,7 @@ def get_result(self, cursor=None, limit=100):
         )
 
         stop = offset + limit + extra
-        results = (
-            list(combined_querysets[offset:stop])
-            if self.using_dates
-            else list(combined_querysets[: (limit + extra)])
-        )
+        results = list(combined_querysets[offset:stop])
```

Comment:
Why stop vs extra?

Reply:
`stop` is `offset + limit + extra`, this was changed accidentally by Scott last week and it caused some issues (paging with an offset), so I am changing it back.